armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -212,10 +213,19 @@ def _compareAssem(candidate, current):
     @staticmethod
     def _getParamMax(a, paramName, blockLevelMax=True):
         """Get parameter with Block-level maximum."""
+        volIntegratedParamNames = [
+            p.name
+            for p in a.getBlocks()[0].p.paramDefs.atLocation(
+                ParamLocation.VOLUME_INTEGRATED
+            )
+        ]
+        multiplier = (
+            a.getSymmetryFactor() if paramName in volIntegratedParamNames else 1.0
```

Comment:
Perhaps we could further simplify this logic by first checking if `a.getSymmetryFactor()` is 1?

```python
multiplier = a.getSymmetryFactor()
if multiplier != 1.0:
    # your logic above to calculate volIntegratedParamNames
    multiplier = multiplier if paramName in volIntegratedParamNames else 1.0
```

Does that make sense?  I just mean, if the symmetry factor is 1.0, we don't need to do any of this logic, and we can skip a lot of math by not inspecting parameters.